lambda/index.ts

```diff
@@ -157,6 +165,36 @@ function updateDocumentAddTags(event: Event): Promise<Event> {
   });
 }
 
+const pascalizeKeys = (obj): any => {
+  const isObject = o => Object.prototype.toString.apply(o) === '[object Object]'
+  const isArray = o => Object.prototype.toString.apply(o) === '[object Array]'
+
+  let transformedObj = isArray(obj) ? [] : {}
+
+  for (let key in obj) {
+    // replace the following with any transform function
+    const transformedKey = toPascalCase(key)
+
+    if (isObject(obj[key]) || isArray(obj[key])) {
+      transformedObj[transformedKey] = pascalizeKeys(obj[key])
+    } else {
+      transformedObj[transformedKey] = obj[key]
+    }
+  }
+  return transformedObj
+}
+
+function toPascalCase(string) {
+  return `${string}`
+    .replace(new RegExp(/[-_]+/, 'g'), ' ')
+    .replace(new RegExp(/[^\w\s]/, 'g'), '')
+    .replace(
+      new RegExp(/\s+(.)(\w*)/, 'g'),
+      ($1, $2, $3) => `${$2.toUpperCase() + $3.toLowerCase()}`
+    )
+    .replace(new RegExp(/\w/), s => s.toUpperCase());
+}
```

Comment:
Nested objects are not pascal-ized when using `pascalCaseProperties: true,` on the CustomResource. So doing that work here. 

Was torn on if this should be before the CustomResource is called or in the lambda (where it is currently) Open to suggestions.